fuse_optimizers/src/optimizer.cpp

```diff
@@ -123,64 +109,83 @@ Optimizer::~Optimizer()
 
 void Optimizer::loadMotionModels()
 {
-  // Read and configure all of sensor model plugins
-  if (!private_node_handle_.hasParam("motion_models"))
-  {
-    return;
+  const std::string param_prefix = "motion_models.";
+  const std::string list_param_full_name = param_prefix + "motion_model_list";
```

Comment:
I think we can avoid adding the list parameters, which makes the parameter interface a little closer to the original. I opened a PR on your fork, mind taking a look?

https://github.com/methylDragon/fuse/pull/5